lib/language.js

```diff
@@ -4108,206 +4115,242 @@ function init() {
       ,nb: 'Nyeste først'
       ,ro: 'mai noi primele'
       ,bg: 'Новите най-отгоре'
+      ,it: 'più recente in alto'
       }
     ,'All sensor events' : {
       cs: 'Všechny události sensoru'
       ,nb: 'Alle sensorhendelser'
       ,ro: 'Evenimente legate de senzor'
       ,bg: 'Всички събития от сензора'
+      ,it: 'Tutti gli eventi del sensore'
       }
     ,'Remove future items from mongo database' : {
       cs: 'Odebrání položek v budoucnosti z Mongo databáze'
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1190556461/issues/source?bid=2452379&fileBranchId=2923644#l4128)